Lesson 2 - Functions/code/exercise2/code.js

```diff
@@ -6,9 +6,13 @@
 	}
 
 	mapArray = function (array, callback, ctx) {
-        var newArr = [];
+        var newArr = [10,11,12];
```

Comment:
You don't use any of functions parameters :disappointed: 

Idea is to take context - `ctx` into account.